cpu/sam0_common/periph/flashpage.c

```diff
@@ -68,7 +68,7 @@ void flashpage_write_raw(void *target_addr, void *data, size_t len)
             ((unsigned)data % FLASHPAGE_RAW_ALIGNMENT)));
 
     /* ensure the length doesn't exceed the actual flash size */
-    assert(((uint8_t*)target_addr + len) <
+    assert(((unsigned)target_addr + len) <
```

Comment:
shouldn't this be uintptr_t?

Reply:
Well, this is another possibility.

Reply:
I made it consistent with other `asserts` in this file, so either leave as is (which works) or change all `assert` to use `uintptr_t`. But I guess than we can grep all-over RIOT to fix that.

Reply:
ok